uber/automated_emails.py

```diff
@@ -1,28 +1,30 @@
-### WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
-#   Note that since the EVENT_NAME is used in most of these emails, changing the event name mid-year
-#   could cause literally thousands of emails to be re-sent!
+# WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
+# Note that since c.EVENT_NAME is used in most of these emails, changing the event name mid-year
+# could cause literally thousands of emails to be re-sent!
 
 from uber.common import *
 
+
 class AutomatedEmail:
     instances = OrderedDict()
 
```

Comment:
Here's the basic idea:

We have a bunch of background tasks.  Let's say we wrote two different background tasks slightly differently.

```python
from uber.common import *

def foo(): print('Hello World')
if c.DO_STUFF:
    DaemonTask(foo, interval=60)


def bar():
    if c.DO_STUFF:
        print('Hello Kitty')
DaemonTask(bar, interval=60)
```

Both of the above will print a message every 60 seconds if ``c.DO_STUFF`` is true.  However, in the former case, if the value of ``c.DO_STUFF`` was changed at runtime from ``False`` to ``True``, the background task would never have started in the first place, so it wouldn't begin printing things.  The latter case always starts the background task, and when ``c.DO_STUFF`` is ``False`` it just skips doing anything, so if that config option changes at runtime it will start doing it's thing without the server needed to be restarted.

This is the type of refactor we need to make everywhere that we want to use configuration that can change if/when we need want to support something like Redis where we intend to change config options while the server is running.